ci/goreleaser/goreleaser-el7.yml

```diff
@@ -136,15 +115,13 @@ publishers:
 archives:
   - format: binary
     allow_different_binary_count: true
-
 dockers:
-
 # Build plugin-compiler
 - ids:
     - std
   image_templates:
     - tykio/tyk-plugin-compiler:{{ .Tag }}-el7
-    - "tykio/tyk-plugin-compiler:v{{ .Major }}.{{ .Minor }}{{.Prerelease}}-el7"
+    - tykio/tyk-plugin-compiler:v{{ .Major }}.{{ .Minor }}{{.Prerelease}}-el7
```

Comment:
There's a `-` missing here from .Prerelease (`rc15`), I've recently suggested that maybe we can use {if} checks or something to ensure a valid semver tag: `v{{.Tag}}` is valid for the whole release, but we build `v4.0rc16` (invalid) instead of `v4.0-rc16` (valid). Please check other parts where .Prerelease may be used;

As it's not a part of of TD-918 it's up to you to address this, it can be a separate jira ticket to be tracked.

If it can be addressed, then this problem repeats whereever `.Prerelease` is used (a couple of places)

Reply:
Yes, we did try something for this - but hasn't been able to fix, yet. Will track this in a separate ticket and address.